apps/server/src/employees/employees.controller.ts

```diff
@@ -131,7 +131,7 @@ export class EmployeesController {
   }
 
   @UseGuards(JwtAuthGuard)
-  @Get('me')
+  @Get('find/me')
```

Comment:
what's the reasoning behind adding the `/find`? should this instead describe what you're getting?

Reply:
when I set it as just `/me`, the routes would confuse `me` as `{id}`, so it would call `GET` on `/api/employees{id}` instead of `/me`.